pages/linux/eject.md

```diff
@@ -0,0 +1,31 @@
+# eject
+
+> Eject cds, floppy disks and tape drives.
+
+- Display the default device:
+
+`eject -d`
+
+- Eject the default device:
+
+`eject`
+
+- Eject a specific device:
```

Comment:
In this case, what does it eject ? CD, floppy or tape ? We should mention that.

Reply:
It's ordered as:
`cd-rom, scsi, floppy and tape`

Reply:
Let's mention that in parentheses.